docs/Milestones.md

```diff
@@ -0,0 +1,47 @@
+# Milestones
+
+#### [M0] Milestone 0: **Diseño del modelo conceptual y viabilidad técnica**
+
+**Producto**:  
+- **Diagrama técnico detallado** que especifique las fuentes de datos y su integración en la plataforma.
```

Comment:
Por favor, no pidáis cosas aleatorias que no son útiles para el proceso de desarrollo.